### PR TITLE
Add first part of DataPager articles.

### PR DESCRIPTION
**Pull request by didiyordanova**

### CODE REVIEW

File: controls/datapager/styling/buttons.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+<snippet id='datapager-styling-buttons' />
+
+**2.** Add the following namespace:
+
+```XAML
+xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
+```
+
+**3.** Define the buttons resources in the page's resource:
```
Comment (yordan-mitev) [inline controls/datapager/styling/buttons.md:51]:
```suggestion
**3.** Define the button resources in the page's resource:
```

File: controls/datapager/styling/navigationview.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+title: Navigation View Styling
+page_title: .NET MAUI DataPager Documentation - NavigationView Styling
+description: Learn what are the styling options for the Telerik UI for .NET MAUI DataPager NavigationView view.
+position: 2
+slug: datapager-styling-navigationview
+---
+
+# Styling the .NET MAUI DataPager Navigation View
+
+The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the navigation view that presents the number of pages and the combo that allows you to select different page, by setting the `NavigationViewStyle` (`Style` with target type of `telerik:DataPagerNavigationView`) property.
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:11]:
```suggestion
The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the navigation view, which presents the number of pages and the combo box for selecting different pages.
```

File: controls/datapager/styling/navigationview.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+The `NavigationViewStyle` uses as a target type the `telerik:DataPagerNavigationView`. The `DataPagerNavigationView` exposes the following properties:
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:13]:
```suggestion
To style the Navigation View, utilize the `NavigationViewStyle` property with a target type of `telerik:DataPagerNavigationView`). 

The `DataPagerNavigationView` exposes the following properties:
```

File: controls/datapager/styling/navigationview.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+* `DataPager` (`RadDataPager`)&mdash;Specifies the associated `Telerik.Maui.Controls.RadDataPager`.
+* `LabelStyle` (`Style` with target type `Label`)&mdash;Specifies the style which applies to the label.
+* `ComboBoxStyle` (`Style` with target type `RadComboBox`)&mdash;Specifies the associated `Telerik.Maui.Controls.RadDataPager`.
+* `BackgroundColor` (`Color`)&mdash;Specifies the background color of the page sizes view.
+
+Here is an example.
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:20]:
```suggestion
Here is an example demonstrating how to use the `NavigationViewStyle` property.
```

File: controls/datapager/styling/navigationview.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+Define the DataPager in XAML:
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:22]: Use a list for the steps.

File: controls/datapager/styling/navigationview.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+</Style>
+<Style x:Key="NavigationViewStyle" TargetType="telerik:DataPagerNavigationView">
+    <Setter Property="ComboBoxStyle" Value="{StaticResource ComboStyle}" />
+    <Setter Property="LabelsStyle" Value="{StaticResource LabelStyle}" />
+</Style>
+```
+
+## Styling the Navigation ComboBox View
+
+The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the view that allows you to navigate to the pages by using the `NavigationComboBoxStyle` (`Style` with target type of `telerik:RadComboBox`) property.
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:45]:
```suggestion
To customize the appearance of the ComboBox view that allows the navigation to a specific page, use the `NavigationComboBoxStyle` (`Style` with target type of `telerik:RadComboBox`) property.
```

File: controls/datapager/styling/navigationview.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+The `NavigationComboBoxStyle` uses as a target type the `RadComboBox` control. Use the styling properties described in the [ComboBox Styling]({%slug combobox-styling%}) article to style the combobox part of the DataPager control.
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:47]:
```suggestion
The target type of the `NavigationComboBoxStyle` is the `RadComboBox` control. Use the styling properties described in the [ComboBox Styling]({%slug combobox-styling%}) article to style the ComboBox part of the DataPager control. 
```

File: controls/datapager/styling/navigationview.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+</Style>
+```
+
+## Styling the Navigation ComboBox View
+
+The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the view that allows you to navigate to the pages by using the `NavigationComboBoxStyle` (`Style` with target type of `telerik:RadComboBox`) property.
+
+The `NavigationComboBoxStyle` uses as a target type the `RadComboBox` control. Use the styling properties described in the [ComboBox Styling]({%slug combobox-styling%}) article to style the combobox part of the DataPager control. 
+
+Here is an example:
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:49]:
```suggestion
Here is an example demonstrating how to use the `NavigationComboBoxStyle` property.
```

File: controls/datapager/styling/pagesizes.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+title: PageSizes Styling
```
Comment (yordan-mitev) [inline controls/datapager/styling/pagesizes.md:2]:
```suggestion
title: Page Sizes Styling
```

File: controls/datapager/styling/pagesizes.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+page_title: .NET MAUI DataPager Documentation - PageSizes Styling
```
Comment (yordan-mitev) [inline controls/datapager/styling/pagesizes.md:3]:
```suggestion
page_title: .NET MAUI DataPager Documentation - Page Sizes Styling
```

File: controls/datapager/styling/pagesizes.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the view that allows you to navigate to the page sizes by using the `PageSizesViewStyle` (`Style` with target type of `telerik:DataPagerPageSizesView`) property.
+
+The `telerik:PageSizesViewStyle` uses as a target type the `telerik:DataPagerPageSizesView`. The `DataPagerPageSizesView` exposes the following properties:
+
+* `DataPager` (`RadDataPager`)&mdash;Specifies the associated `Telerik.Maui.Controls.RadDataPager`.
+* `LabelStyle` (`Style` with target type `Label`)&mdash;Specifies the style which applies to the label.
+* `ComboBoxStyle` (`Style` with target type `RadComboBox`)&mdash;Specifies the associated `Telerik.Maui.Controls.RadDataPager`.
+* `BackgroundColor` (`Color`)&mdash;Specifies the background color of the page sizes view.
+
+Here is an example.
```
Comment (yordan-mitev) [inline controls/datapager/styling/pagesizes.md:20]:
```suggestion
Here is an example demonstrating how to use `PageSizesViewStyle`.
```

File: controls/datapager/styling/pagesizes.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+Define the DataPager in XAML:
```
Comment (yordan-mitev) [inline controls/datapager/styling/pagesizes.md:22]: Use a list.

File: controls/datapager/page-configuration.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+By using the `PageSizes` (`IList<int>`) you can specify the pages that the end user can choose from.
+
+
+## Current Page
+
+The DataPager manages the current page depending on the user actions. In addition you can specify the current index of the DataPager by using the `PageIndex` (`int`) property.
+The property contains the index of the currently selected page. The default value is `-1`.
+
+Here is an example with the `PageIndex` set:
```
[the remark this answers is not in the document]
Comment (didiyordanova) [reply controls/datapager/page-configuration.md:26]: no example for this

[the remark this answers is not in the document]
Comment (didiyordanova) [reply controls/datapager/page-configuration.md:26]: removed the line

File: controls/datapager/page-configuration.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+
+# .NET MAUI DataPager Page Configuration
+
+This article explains the configuration options you can apply to the pages inside the DataPager control.
+
+## Page Size
+
+The DataPager splits the data into separate pages with a certain size. To specify the size of the page you have to use the `PageSize` (`int`) property. The default value is `10`.
+
+By using the `PageSizes` (`IList<int>`) you can specify the pages that the end user can choose from.
```
[the remark this answers is not in the document]
Comment (didiyordanova) [reply controls/datapager/page-configuration.md:18]: there is an example in the article

File: controls/datapager/styling/navigationview.md [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+description: Learn what are the styling options for the Telerik UI for .NET MAUI DataPager NavigationView view.
+position: 2
+slug: datapager-styling-navigationview
+---
+
+# Styling the .NET MAUI DataPager Navigation View
+
+The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the navigation view, which presents the number of pages and the combo box for selecting different pages.
+
+To style the Navigation View, utilize the `NavigationViewStyle` property with a target type of `telerik:DataPagerNavigationView`).
```
Comment (yordan-mitev) [inline controls/datapager/styling/navigationview.md:13]:
```suggestion
To style the Navigation View, utilize the `NavigationViewStyle` property (of type `Style` with a target type of `telerik:DataPagerNavigationView`). 
```

**Review by yordan-mitev — APPROVED** [review]
Consider rewriting the definitions of  `PageSize` and `PageSizes` [here](https://github.com/telerik/maui-docs/pull/987#discussion_r1832545197).

**Pull request merged**